docs/users_guide/glasgow_exts.rst

```diff
@@ -12450,7 +12455,6 @@ That being said, with the appropriate use of wrapper datatypes, the
 above limitations induce no loss of generality: ::
 
     {-# LANGUAGE ConstraintKinds           #-}
-    {-# LANGUAGE DeriveDataTypeable        #-}
```

Comment:
With the `Typeable` derivation removed, this example code no longer needs the `DeriveDataTypeable` extension.